distributions/distribution-resources/src/main/resources/conf/icons/classic/readme.txt

```diff
@@ -0,0 +1,8 @@
+
+openHAB2 conf/icons/classic folder
```

Comment:
I would remove this first line in all the files - note that the folder name can be e.g. different for an apt-get installation, so this can be rather confusing.